clio-server/config/clio-logback.xml.ctmpl

```diff
@@ -4,15 +4,14 @@
     <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
         <file>{{$log_dir}}/clio.log</file>
 
-        <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
-            <fileNamePattern>{{$log_dir}}/clio.%i.log.gz</fileNamePattern>
-            <minIndex>1</minIndex>
-            <maxIndex>5</maxIndex>
-        </rollingPolicy>
+        <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+            <!-- roll over daily, inferred by logback from the filename pattern -->
+            <fileNamePattern>{{$log_dir}}/clio-%d{yyyy-MM-dd}.log</fileNamePattern>
 
-        <triggeringPolicy class="ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy">
-            <maxFileSize>10MB</maxFileSize>
-        </triggeringPolicy>
+            <!-- keep 30 days' worth of logs, up to 5GB -->
+            <maxHistory>30</maxHistory>
+            <totalSizeCap>5GB</totalSizeCap>
```

Comment:
I worry about any log size cap at all until we have an idea about how much space 30 days of logs should take up. On the other hand, if logs are compressed we will probably be fine.